src/main/java/org/apache/brooklyn/cloudfoundry/location/CloudFoundryPaasLocation.java

```diff
@@ -387,4 +408,224 @@ public int getMemory(String applicationName) {
         return getApplication(applicationName).getMemoryLimit();
     }
 
+    public void createServiceInstance(Map<?, ?> params) {
+        ConfigBag serviceSetUp = ConfigBag.newInstance(params);
+        String serviceName = serviceSetUp.get(VanillaCloudFoundryService.SERVICE_NAME);
+        checkArgument(Strings.isNonBlank(serviceName), "Service Name can not be blank");
+        String instanceName =
+                serviceSetUp.get(VanillaCloudFoundryService.SERVICE_INSTANCE_NAME);
+        checkArgument(Strings.isNonBlank(instanceName), "Service Instance Name can not be blank");
+        String plan = serviceSetUp.get(VanillaCloudFoundryService.PLAN);
+        checkArgument(Strings.isNonBlank(plan), "Plan can not be blank");
+
+        try {
+            getClient().services()
+                    .createInstance(CreateServiceInstanceRequest.builder()
+                            .serviceName(serviceName)
+                            .serviceInstanceName(instanceName)
+                            .planName(plan)
+                            .build())
+                    .doOnSuccess(v ->
```

Comment:
I do not understand that, lambda writes in the log, as the rest of location's operations